CodingTracker.jwhitt3r/CodingTracker.jwhitt3r/DatabaseManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Data.Sqlite;
+
+namespace CodingTracker.jwhitt3r
+{
+    /// <summary>
+    /// Ensures that a database table is correctly setup before any interaction has occurred
+    /// </summary>
+    internal class DatabaseManager
+    {
+        internal void CreateTable(string connectionString)
+        {
+            using (var connection = new SqliteConnection(connectionString))
+            {
+                using (var tableCmd = connection.CreateCommand())
+                {
+                    connection.Open();
+                    tableCmd.CommandText =
+                        @"CREATE TABLE IF NOT EXISTS coding (
+                            Id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
I'm getting the following error, I assume because of this query: 

![image](https://user-images.githubusercontent.com/108711476/231960574-c8efefc3-ab25-4c2a-a3dc-7e5a4603f011.png)